src/Components/FeaturedProjects.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable implicit-arrow-linebreak */
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { TagContainer, Tag } from 'Styled components/TagStyles'
+import { OuterWrapper, InnerWrapper, P, A } from '../Styled components/GlobalStyles'
+import { WhitePinkH2, PinkH3 } from '../Styled components/HeadlineStyles'
+
+const GitHubAPI = 'https://api.github.com/users/jessikalind/repos'
+
+const FeaturedProjects = () => {
+  const [projects, setProjects] = useState([])
+
+  useEffect(() => {
+    fetch(GitHubAPI)
+      .then((res) => res.json())
+      .then((data) => setProjects(data))
+  })
+
+  // Filtering out the projects I do not want to include in the Featured projects-section
+  const featuredProjects = projects.filter((project) => project.name.includes('happy-thoughts') || (project.name.includes('movies')) || (project.name.includes('survey')) || (project.name.includes('music')))
```

Comment:
When I read the instruction its confusing, these are the projects that ARE included in the Featured project section? But other than that this whole component is very nicely structered and as always, nicea clean code.